### PR TITLE
Using goimports on .go files under services directory.

### PR DESCRIPTION
**Pull request by vijtrip2**
Issue #, if available:

Description of changes:
Based on the comments from varun1524 on pull requset #57 
* Applying goimports on the files under service directory for bookstore & petstore services.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by vijtrip2** [comment]
Please update the commit message to "Using goimports on .go files under services directory." when merging the pull request.

**Comment by vijtrip2** [comment]
I added some more dependency fix as suggested here https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-505627280 for successful run of integration-tests.

File: test/integration/services/common.go [hunk 1]
```diff
@@ -10,6 +10,11 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+const (
+	defaultHost            = "http://127.0.0.1:8080"
+	integrationTestAppName = "integration-test-app"
```
Comment (varun1524) [inline test/integration/services/common.go:15]: thanks for fixing it. 
integrationTestAppName is unused if you can remove it.

Comment (vijtrip2) [reply test/integration/services/common.go:15]: Yeah. I just let that variable stay since this file will be developed more and it can be omitted later.

**Pull request merged**